align_system/algorithms/oracle_adm.py

```diff
@@ -21,7 +21,8 @@ def __init__(self, misaligned: bool=False, probabilistic: bool=False):
         self.choice_history = {} # Used for cumulative KDE alignment
 
     def choose_action(self, scenario_state, available_actions, alignment_target,
-                      distribution_matching='sample', kde_norm='rawscores', **kwargs):
+                      distribution_matching='sample', kde_norm='rawscores',
+                      norm_factor=0.5, **kwargs):
```

Comment:
Just for my understanding, the `norm_factor` here only applies when the `kde_norm` value is `"priornorm"` is that right?  Just want to make sure that normalization isn't being applied by default.

Reply:
I'll change it quick

Reply:
Thanks for making that change, I'll go ahead and merge in a few unless there's anything still outstanding from your end?

Reply:
I think it's good to go!